docs/explanations/how_to_interpret_output_of_fix_plan_api.md

```diff
@@ -73,6 +75,8 @@ If we take a look at a CVE object, we will see the following structure:
 * **warnings**: Warnings that can happen when generating the plan. For example, if the CVE doesn't
   have a fix available for a given package, this will be reflected here. We will detail the possible
   warnings in a following section.
+* **additional_data**: Additional data for the CVE being fixed. Currently, this field is only
```

Comment:
I'm a bit confused about this bit. The "additional data" field doesn't appear in the USN section at the bottom of the page, so if it's only found under the CVEs enpoint but only used by USNs then I'm not sure how that should work? Surely it would never come up? Why aren't Launchpad bugs also reported for the CVEs?